blog/_posts/2022-09-06-ads-object-search.md

```diff
@@ -0,0 +1,91 @@
+
+```
+Aldebaran collection:astronomy
+```
+
+[^1]:A query field in general has the format of a field name, directly followed by a colon (like e.g. `author:`). The result of a fielded query is that only specific data types in our database are queried with the data provided (see [this page](https://ui.adsabs.harvard.edu/help/search/comprehensive-solr-term-list) in our online Help for a comprehensive list of query fields)
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/ads-object-search_f1.png" />
+<em>Figure 1</em>
+</div>
+<br>
+
+Before going into specifics of search technicalities, let's first explore the question of the necessity of adding the collection filter (restricting the query to the ADS Astronomy collection). The ADS holdings cover a wide range of disciplines; this is the result of indexing multidisciplinary publications like Science, Nature and arXiv e-prints, but also because we include the literature that is cited by and which cites the core literature. This is an important fact to keep in mind when exploring the ADS holdings. Often, names of astronomical and planetary objects are taken from areas like mythology. It also happens that, for example, projects, services or companies, unrelated to astronomy, are named after astronomical objects (like Aldebaran Robotics). Records not fetched by the query above are retrieved by the query ```Aldebaran NOT collection:astronomy```. Inspection of the results show publications that are definitely related to the star Aldebaran. As illustrated in the screenshot (Figure 1), it helps to check the highlights for the context of the matches. For example, the arXiv record [The Ancient Astronomy of Easter Island: Aldebaran and the Pleiades](https://ui.adsabs.harvard.edu/abs/2016arXiv161008966R/abstract) (in the arXiv category `Physics - History and Philosophy of Physics`) and the SPIE record [Measurement of the speed of light from extraterrestrial sources](https://ui.adsabs.harvard.edu/abs/2015SPIE.9608E..0EW/abstract). There are also records that are obviously unrelated to the star (e.g. humanoid robots, Aldebaran Sandstone). In the ADS curation workflow we attempt to classify records from multidisciplinary sources (like the journals Science and Nature). In some cases it may be better to optimize your query beforehand (by adding filters), while in other cases it will be better to cast a wide net first and apply filters afterwards. The best strategy depends on one's goals.
```

Comment:
This sentence: `For example, the arXiv record...` seems to be missing its second half. Do you mean something like `For example, the arXiv record... appear to be related to the star Aldebaran.`